packages/salesforcedx-utils-vscode/src/telemetry/utils/devModeUtils.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as Settings from '../../../src/settings';
+
+export const checkDevLocalLogging = (extName: string) => {
+  const isLocalLogging = Settings.SettingsService.isAdvancedSettingEnabledFor(
+    extName,
+    Settings.AdvancedSettings.LOCAL_TELEMETRY_LOGGING
+  );
+  return isLocalLogging;
```

Comment:
``` suggestion
  return  Settings.SettingsService.isAdvancedSettingEnabledFor(
    extName,
    Settings.AdvancedSettings.LOCAL_TELEMETRY_LOGGING
  );
```

Reply:
I added the variable to make it easier to understand at quick glance what that was checking for, however I believe changing the method name will be a better indicator.